src/mock/ray/gcs/gcs_server/gcs_placement_group_manager.h

```diff
@@ -11,17 +11,12 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-#include "ray/gcs/gcs_server/gcs_placement_group_mgr.h"
 
-namespace ray {
-namespace gcs {
+#pragma once
```

Comment:
I wonder if we could get a clang-tidy rule for pragma once.  I'm a little unclear on cases where we might not want to have it in ray